Unity-Technologies/Graphics
# Fix LensFlare occlusion on scene view

Fix flickering of LensFlare Occlusion with TAA (so HDRP only).
By doing accumulation "unanimity vote" through multiple frame (8 by default)
Before this PR:
![1LGEzSRXeS](https://user-images.githubusercontent.com/55133890/134497709-65614615-eabe-4a78-b3ff-740a49f888e8.gif)
With the PR:
![Inszu7cA68](https://user-images.githubusercontent.com/55133890/134497736-6ff0d0d7-ba0f-4b18-b68c-31d5a2f97fb1.gif)

QA:
- Flickering occurs before when the LensFlare had occlusion enable and place very close to an edge with TAA enabled
- Should extended testing with various parameter of TAA to see if the flickering is fixed
- Max-Lens-Flare-With-Occlusion: supported 128. Variables available in the code LensFlareCommonSRP.maxLensFlareWithOcclusion;
